rapids/rapids.sh

```diff
@@ -59,10 +54,45 @@ function execute_with_retries() {
   return 1
 }
 
+function install_dask_rapids() {
+  local base
+  base=$(conda info --base)
+  local pinned
+  pinned=${base}/conda-meta/pinned
+  local mamba_env
+  mamba_env=mamba
+  
+  # Using mamba significantly reduces the conda solve-time. Create a separate conda
+  # environment with mamba installed to manage installations.
+  conda create -y -n ${mamba_env} -c conda-forge mamba
+
+  # RAPIDS releases require fixed PyArrow versions. Unpin PyArrow to solve
+  # for new environment.
+  sed -i 's/pyarrow [\.0-9\*]*/pyarrow/g' ${pinned}
+
+  # Install RAPIDS and cudatoolkit. Use mamba in new env to resolve base environment
+  installer=${base}/envs/${mamba_env}/bin/mamba
+  $installer install -y \
```

Comment:
Seems like there no need to introduce local `installer` variable that can be inlined?

Reply:
Ack